### PR TITLE
Code to Generate Random String

### PR DESCRIPTION
**Pull request by gaut2003**

### CODE REVIEW

File: src/randomStringGenerator.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Random String Generator
+Generate random strings using Javascript
+
+## Result
+```
+  50qKj
```
Comment (DevTMK) [inline src/randomStringGenerator.md:6]: should be - 
generateString(5); // 50qkj

File: src/randomStringGenerator.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+## Snip Langauge
+* [JavaScript](#javascript)
+
+### JavaScript
+```js
+const characters ='ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```
Comment (DevTMK) [inline src/randomStringGenerator.md:14]: Add this inside the function

File: src/randomStringGenerator.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+function generateString(length) {
+    let result = ' ';
```
Comment (DevTMK) [inline src/randomStringGenerator.md:17]: Remove the space, it's changing the desired output

File: src/randomStringGenerator.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    const charactersLength = characters.length;
+    for ( let i = 0; i < length; i++ ) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+
+    return result;
+}
+
+console.log(generateString(5));
```
Comment (DevTMK) [inline src/randomStringGenerator.md:26]: Remove this console log, not needed

**Review by DevTMK — CHANGES_REQUESTED** [review]

**Comment by DevTMK** [comment]
Merging, Thanks

**Comment by gaut2003** [comment]
Thank you. 
Hey I am new to Open source. Can you help me in my journey to open source.

**Comment by DevTMK** [comment]
> Thank you. Hey I am new to Open source. Can you help me in my journey to open source.

Sure mate

**Comment by DevTMK** [comment]
You stuck on something?

**Comment by gaut2003** [comment]
So, how can I contact you for further queries and doubts?

**Comment by DevTMK** [comment]
Are you on discord? You can contact me there

**Comment by gaut2003** [comment]
Ya I am. Your id or smthng?

**Pull request merged**